sdcm/nemesis.py

```diff
@@ -4735,6 +4738,30 @@ def _disrupt_toggle_audit(self, store: AuditStore):
         if errors:
             raise AuditLogTestFailure("\n".join(errors))
 
+    @staticmethod
+    @contextlib.contextmanager
+    def nodetool_wrap(node, start_command, end_command):
```

Comment:
I would move it to somewhere else

And document it a bit what the purpose of it

Also it's a context manger not a wrapper/ decorator, so name might have context in it.

It would be nice if it would yield the start command output

Reply:
> I would move it to somewhere else

Would sdcm/utils/decorators.py be a fitting location?

> And document it a bit what the purpose of it
> 
> Also it's a context manger not a wrapper/ decorator, so name might have context in it.

Correct!

> It would be nice if it would yield the start command output

Good idea, will do!

Reply:
Done and done